scripts/prepare/addon-bundle.ts

```diff
@@ -96,7 +96,7 @@ const run = async ({ cwd, flags }: { cwd: string; flags: string[] }) => {
 
   if (reset) {
     await fs.emptyDir(OUT_DIR);
-    await fs.emptyDir(metafilesDir);
+    // await fs.emptyDir(metafilesDir);
```

Comment:
Was this commented out for testing purposes, or can it be removed?

Reply:
This file will get deleted anyway, but the problem with this line, is that this clears all the metafiles, not just for the one package the bundle script was called for.